src/utils/resolveImport.ts

```diff
@@ -11,13 +9,15 @@ export default function resolveImport({
     fileLocation: string;
     location: string;
 }) {
-    let relativePath = relative(dirname(fileLocation), resolve(baseDir, location));
+    const relativePath = relative(dirname(fileLocation), resolve(baseDir, location)).replace(/\\/g, "/");
```

Comment:
TBH it's kinda hard for me to understand what we're doing here and why.
AFAIR on *nix it generates posix path, and on windows it generates win path and then it converts it back to posix? But why? What's the output for those two queries? How are they different?
```relative(posix.dirname(fileLocation), posix.resolve(baseDir, location))```
and
```relative(win32.dirname(fileLocation), win32.resolve(baseDir, location)).replace(/\\/g, "/")```